intrahospital_api/loader.py

```diff
@@ -104,7 +104,7 @@ def create_rfh_patient_from_hospital_number(hospital_number, episode_category, r
         start=datetime.date.today()
     )
 
-    if run_async is None:
+    if run_async is not None:
```

Comment:
Doesn't this run it synchronously if the spelling is `create_rfh_patient_from_hospital_number('213', 'infection', run_async=False)`
That also feels like not the right thing to do

Reply:
Why doesn't it just pass through the parameter whatever the value, why the is not None check?